### DISCUSSION TITLE
Is it possible to auto-minify, then push to another branch?

### DISCUSSION THREAD

**Discussion by qwbarch** [nizarmah/auto-minify#24] [Q&A] [answered]
I have a repository where ``main`` holds the latest commits, while I want my other branch called ``minified`` to hold the minified version of the main branch.
Would it be possible for ``auto-minify`` to do this in a github action?

**Comment by nizarmah** [comment, marked as answer]
Hello @qwbarch! Sorry for the late reply! 

Yes, it's possible to push the minified files to the `minified` branch. 

Before I share the code on how to do it, I'll share the logic behind it. 

There are three main steps for using this action:
1. Checking out the files from a specific repository and branch
1. Running the `auto-minify` action
1. Committing the files to a specific repository and branch

Those three steps are broken into the following segments, in the code:
```docker
steps:
  # Checking out the files from a specific repository and branch
  - uses: actions/checkout@v2

  # Running the `auto-minify` action
  - name: Auto Minify
    uses: nizarmah/auto-minify@v2.1

  # Committing the files to a specific repository and branch 
  - name: Auto committing minified files
    uses: stefanzweifel/git-auto-commit-action@v4
    with:
      commit_message: "Github Action: Auto Minified JS and CSS files"
      branch: "minified"  # <--- change this line to your branch name or use "${{ github.ref }}" for default branch
```
